src/cata_tiles.cpp

```diff
@@ -593,6 +598,8 @@ void tileset_cache::loader::load_tileset( const cata_path &img_path, const bool
     const int expected_tilecount = ( tile_atlas->w / sprite_width ) *
                                    ( tile_atlas->h / sprite_height );
     extend_vector_by( ts.tile_values, expected_tilecount );
+    extend_vector_by( ts.monochrome_tile_values, expected_tilecount );
+    extend_vector_by( ts.greenchrome_tile_values, expected_tilecount );
```

Comment:
This looks like it duplicates all of the textures in the tileset, meaning this increases the memory requirement of tilesets by 40%. Can you profile what that looks like for our tilesets? How much more memory will this feature require?